.evergreen.yml

```diff
@@ -85,6 +85,7 @@ functions:
           export HADRON_METRICS_BUGSNAG_KEY="${metrics_bugsnag_key}";
           export HADRON_METRICS_INTERCOM_APP_ID="${metrics_intercom_app_id}";
           export HADRON_METRICS_STITCH_APP_ID="${metrics_stitch_app_id}";
+          export HADRON_METRICS_SEGMENT_API_KEY="${metrics_segment_api_key}";
```

Comment:
Should we continue to use `HADRON_` as a name?

Reply:
I know you already brought this up in the ticket – I don’t think it matters much, I agree that it’s not really ideal, but I’d like to keep this change focused on really just enabling telemetry. We can change the prefix to `COMPASS` for all of these variables as a normal code cleanup thing.

Reply:
Ah you mean we rename everything? I'm totally fine keeping it for now in any case, my point was that the other variables will just go away with the old telemetry so there was no reason to keep it consistent

Reply:
Only the stitch one will go away, though – so yeah, I’d either rename everything or nothing :)